utils/details.js

```diff
@@ -105,7 +107,9 @@ class ActionDetails {
     })
 
     const actions = []
-    const actionRegEx = /- ([A-Z0-9-]+)\/([A-Z0-9-_]+|\*)(@.*){0,1}/i
+
+    const actionRegEx = /- ([a-z0-9-]+)\/([a-z0-9-_]+|\*)(@.*){0,1}/i
+    const dockerRegEx = /- docker:\/\/([a-z0-9-.]+)\/?(([a-z0-9_-]\/?)+)(:(.*)|@([a-z0-9]+:[a-z0-9]+))?/
```

Comment:
Should bother upper and lower be included?  If so, would it be easier switching to `\w` while including the other punctionation?

```suggestion
    const actionRegEx = /- ([a-zA-Z0-9-]+)\/([a-zA-Z0-9-_]+|\*)(@.*){0,1}/i
    const dockerRegEx = /- docker:\/\/([a-zA-Z0-9-.]+)\/?(([a-zA-Z0-9_-]\/?)+)(:(.*)|@([a-zA-Z0-9]+:[a-zA-Z0-9]+))?/
```

Reply:
Good catch, @andyfeller 

Should have been 
```js
const dockerRegEx = /- docker:\/\/([a-z0-9-.]+)\/?(([a-z0-9_-]\/?)+)(:(.*)|@([a-z0-9]+:[a-z0-9]+))?/i
```
to be case insensitive like the other RegEx.

🐙🐱💖